qemu/tests/win_virtio_driver_update_by_installer.py

```diff
@@ -96,6 +96,14 @@ def change_virtio_media(cdrom_virtio):
             static_ip_address = utils_net.get_guest_ip_addr(session_serial, virtio_nic_mac, os_type='windows')
             if static_ip_address != params["static_ip"]:
                 test.fail("Failed to setup static ip,current ip is %s" % static_ip_address)
+            setup_dns_cmd = params["setup_dns_cmd"] % ifname
+            session_serial.cmd_status(setup_dns_cmd)
+            static_dns_address = utils_net.get_windows_nic_attribute(session_serial, global_switch="nicconfig",
```

Comment:
Hi @wji

Is it possible to get the index value via the ifname/mac_address, it will be better than the hard code.
Thanks.

Reply:
@heywji I understand that the old code did not follow PEP8 perfectly, we could keep them as it is. But when we introduce new lines or touch them when fix issues let's try to improve our code style. please keep each line within 79 characters.